index.html

```diff
@@ -20,12 +20,152 @@
             <div class="defineimage"></div>
             <div class="defineimage1"></div>
         </header>
+        <div class="horizontal_line">
+            <a href="#text1" class="wrap ite1">Hello</a>
+            <a href="#text3" class="wrap ite2">Portfolio</a>
+            <a href="#abouttext" class="wrap ite4">About</a>
+            <a href="#text10" class="wrap ite5">Contact</a>
+        </div>
```

Comment:
You have beautifully managed to align this section into the center. Amazing job. 👏 

- [ ] Make sure the link blue text is removed and it follow the design color.

Reply:
I have fixed the issue and changed the color.